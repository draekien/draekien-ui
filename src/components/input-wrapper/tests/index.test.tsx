/** @jsxRumtime classic */
/** @jsx jsx */
import { jsx } from 'theme-ui';
import { render } from '@testing-library/react';
import InputWrapper from '..';

describe('InputWrapper component', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <InputWrapper id="test" label="test" tooltip="test" />
    );

    expect(container).toMatchSnapshot();
  });
});
