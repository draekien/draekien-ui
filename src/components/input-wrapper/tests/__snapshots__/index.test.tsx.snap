// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWrapper component should match snapshot 1`] = `
<div>
  <div
    class="css-hfz0wm-InputWrapper"
  >
    <label
      class="css-7ojqvb-InputWrapper"
      for="test"
    >
      <div
        class="css-3vy6se-Text"
      >
        test
      </div>
    </label>
    <label
      class="css-7ojqvb-InputWrapper"
      for="test"
    >
      <div
        class="css-3vy6se-Text"
      >
        test
      </div>
    </label>
  </div>
</div>
`;
